packages/notifications/src/utils/useNotificationsContext.ts

```diff
@@ -6,10 +6,9 @@
  */
 
 import { createContext, useContext } from 'react';
+import { Type } from '../types';
 
-export type Hue = 'successHue' | 'warningHue' | 'dangerHue' | 'neutralHue';
-
-export const NotificationsContext = createContext<Hue | undefined>(undefined);
+export const NotificationsContext = createContext<Type | undefined>(undefined);
```

Comment:
[nit] The `props.type` is [required](https://garden.zendesk.com/components/alerts#api), thus should always defined. 

To align the context API with the props', should we specify a default rather than using `undefined`?

```suggestion
export const NotificationsContext = createContext<Type | undefined>('info');
```

Doing so will make things a bit clearer in that file:

https://github.com/zendeskgarden/react-components/pull/1842/files#diff-30a97b0e497db8834422a4c3a59fd93ac89d538cb7fc5a65d34f0f9be4850c73R17

https://github.com/zendeskgarden/react-components/pull/1842/files#diff-30a97b0e497db8834422a4c3a59fd93ac89d538cb7fc5a65d34f0f9be4850c73R29

Reply:
I think you caught a gap in color handling actually. 😅

`Close` is used by both Notifications and Alert, but in the former, `type` isn't required, and so the container [falls back](https://github.com/zendeskgarden/react-components/pull/1842/files#diff-b1e6dc5cf32c92184cebb6274c425a9e471f4e2ed272a0ea024c003eb028da0fR69-R72) to `default` and `subtle` for all its fg/bg/border colors.

So, looking at it again, I think the close icon would need a `default` case for this to be correct, rather than the context forcing the type. Hopefully that makes sense!

Reply:
I think I need to look into this more. I'm suspicious the issue is actually because Alert and Notifications share this view component, but only Alert requires type, so it being `undefined` might be a valid scenario. I'll check it out.

Reply:
OK, I confirmed that in all places where `type` is not required (e.g., subcomponents), there are [fall](https://github.com/zendeskgarden/react-components/pull/1842/files#diff-4cd6e77e1b13972c91aa6d3a6fea266c8e7f7c937204f518eeb7a7b6b9ae61f8R42) ]back](https://github.com/zendeskgarden/react-components/pull/1842/files#diff-30a97b0e497db8834422a4c3a59fd93ac89d538cb7fc5a65d34f0f9be4850c73R40). The rest of the components seem to handle the required prop appropriately.

Reply:
Thanks for clarifying. 💯